shell/store/catalog.js

```diff
@@ -331,7 +336,7 @@ export const mutations = {
 };
 
 export const actions = {
-  async load(ctx, { force, reset } = {}) {
+  async load(ctx, { force, reset, storeName } = {}) {
```

Comment:
I had a look for `catalog/load` and couldn't find where this was used, is it still needed?

Reply:
Fixed